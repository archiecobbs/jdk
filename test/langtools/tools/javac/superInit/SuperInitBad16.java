/*
 * @test /nodynamiccopyright/
 * @bug 8193760
 * @summary Test invalid reference to "this" prior to superclass construction
 *
 * @compile/fail/ref=SuperInitBad16.out -XDrawDiagnostics SuperInitBad16.java
 */
class SuperInitBad16 {

    SuperInitBad16() {
        hashCode();
        super();
    }
}
