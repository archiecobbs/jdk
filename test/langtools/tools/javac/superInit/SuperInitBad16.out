SuperInitBad16.java:11:9: compiler.err.cant.ref.before.ctor.called: hashCode()
1 error
